packages/ncids-css/packages/nci-hero/internal/_nci-hero.scss

```diff
@@ -106,3 +106,11 @@
 .nci-hero--with-cta-strip {
 	background: color($theme-color-accent-cool-darker);
 }
+
+.nci-hero__nci-cta-strip-container {
+	background-color: color($theme-color-accent-cool-darker);
```

Comment:
Refer to the comment about moving the CTA strip in the hero variant. By moving the CTA strip inside of `<div class="nci-hero nci-hero--with-cta-strip">`, you won't need that background-color line. The parent div should dictate the color of the CTA strip